openedx/core/djangoapps/credentials/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+edX Platform support for credentials.
+
+This package will be used as a wrapper for interacting with the credentials
```

Comment:
This package will be used as a wrapper for interacting with the credentials service.